validator/sawtooth_validator/state/merkle.py

```diff
@@ -31,6 +31,23 @@
 TOKEN_SIZE = 2
 
 
+def _decode(encoded):
```

Comment:
These should replace the `self._decode` and `self._encode` calls

Reply:
Done. I also went ahead and moved out `_encode_and_hash` for consistency.